DataFormats/Luminosity/interface/PCC.h

```diff
@@ -0,0 +1,56 @@
+#ifndef PCC_PCC_h
+#define PCC_PCC_h
```

Comment:
Also, please rename the file to match the primary class name defined in the file